buttercup/cogs/find.py

```diff
@@ -241,18 +232,87 @@ async def _find(self, ctx: SlashContext, reddit_url: str) -> None:
 
         # If we started with a transcription link, this will be set already.
         if "transcription" not in data:
             data: Dict[str, Dict] = self.get_transcription(data)
 
-        await msg.edit(content="I found the post!", embed=self.to_embed(data))
+        await msg.edit(content="I found the post!", embed=to_embed(data))
 
         if data.get("transcription") or data["submission"].get("claimed_by"):
             data: Dict[str, Dict] = self.get_author(data)
 
-        await msg.edit(content="I found the post!", embed=self.to_embed(data))
+        await msg.edit(content="I found the post!", embed=to_embed(data))
+
+    @cog_ext.cog_slash(
+        name="claimed",
+        description="Find the posts claimed by the given user.",
+        options=[
+            create_option(
+                name="username",
+                description="The user to find the claimed posts for.",
+                option_type=3,
+                required=True,
+            )
+        ],
+    )
+    async def _claimed(self, ctx: SlashContext, username: str) -> None:
+        # Send a first message to show that the bot is responsive.
+        # We will edit this message later with the actual content.
+        msg = await ctx.send(
+            i18n["claimed"]["looking_for_claimed_posts"].format(username)
+        )
+
+        volunteer_response = self.blossom_api.get_user(username)
+        if volunteer_response.status != BlossomStatus.ok:
+            await msg.edit(content=i18n["claimed"]["user_not_found"].format(username))
+            return
+        volunteer = volunteer_response.data
+
+        # We can't specify that no user completed the post yet.
+        # Instead, we ask that the post isn't archived yet.
+        # For a short amount of time this might return completed posts.
+        submission_response = self.blossom_api.get(
+            "submission/", params={"claimed_by": volunteer["id"], "archived": False}
+        )
+        if submission_response.status_code != 200:
+            await msg.edit(content=i18n["claimed"]["posts_not_found"].format(username))
+            return
+        # Filter out completed posts
+        submissions = [
+            post
+            for post in submission_response.json()["results"]
+            if post["completed_by"] is None
+        ]
+
+        if len(submissions) == 0:
+            await msg.edit(content=i18n["claimed"]["no_claimed_posts"].format(username))
+            return
+
+        embed = to_embed(dict(submission=submissions[0], author=volunteer))
+
+        if len(submissions) == 1:
+            await msg.edit(
+                content=i18n["claimed"]["one_post_message"].format(username),
+                embed=embed,
+            )
+            return
+
+        post_list = "\n".join(
+            [
+                i18n["claimed"]["post_list_item"].format(
+                    i + 1, post["tor_url"], post["url"]
```

Comment:
Do we want some pagination/limit here? I can already see a problem with 100 claimed messages flashing before my eyes :P